cmd/orb-server/startcmd/start.go

```diff
@@ -712,13 +712,64 @@ func startOrbServices(parameters *orbParameters) error {
 	return srv.Start(httpServer)
 }
 
-func getProtocolClientProvider(parameters *orbParameters, casClient casapi.Client, casResolver common.CASResolver, opStore common.OperationStore, anchorGraph common.AnchorGraph) (*orbpcp.ClientProvider, error) {
+func getProtocolClientProvider(parameters *orbParameters, casClient casapi.Client, casResolver common.CASResolver,
+	opStore common.OperationStore, anchorGraph common.AnchorGraph,
```

Comment:
We cannot resolve configured origin here since did create request will fail allowed origin check. Please move this code here: https://github.com/trustbloc/orb/blob/f2de6707c6f6cca4222921c192e0da8c9ad2fdd2/pkg/anchor/writer/writer.go#L535

Reply:
Done!